carbon-design-system/carbon-addons-iot-react
# feat(cardeditor): support custom settings panels for all card types, even default ones

Closes #

**Summary**

- Previously only the Custom card types could render custom EditSettingsForm sections.  This adds the capability for all card types

**Change List (commits, features, bugs, etc)**

- feat(CardEditFormContent): show any custom sections if they are included
- fix(TimeSeriesCard): PropTypes for oneOf were incorrect
- fix(CardPropTypes): document the existing renderEditContent prop

**Acceptance Test (how to verify the PR)**

- Verify the new CardEditor story and make sure the custom section shows for the TimeSeriesCard

**Regression Test (how to make sure this PR doesn't break old functionality)**

- Verify that other card types in the CardEditor stories do not render custom content sections
